pandas/_libs/missing.pyx

```diff
@@ -290,12 +291,25 @@ def _create_binary_propagating_op(name, divmod=False):
 
```

Comment:
can `divmod` be changed to `is_divmod`?  dont want to overlap with built-in names

Reply:
Did I define that here or is it called that on master?

Reply:
its like that on master too, still should be changed.  OK for separate PR if thats a hassle